client/src/components/overview/overview.jsx

```diff
@@ -23,11 +34,32 @@ class Overview extends React.Component {
   }
 
   render() {
+    let renderFeatures;
+    let renderStyles;
+    if (this.state.info !== '') {
+      renderFeatures = <Features features={this.state.info.features} />
+      renderStyles = <Styles styles={this.state.styles.results} />
+    } else {
+      renderFeatures = <div></div>
+      renderStyles = <div></div>
+    }
```

Comment:
Great use of conditional rendering here! I think maybe I was trying to use something like this last night and came across componentDidUpdate(). Now I'm wondering if conditional rendering is the better options. What do you think? https://gist.github.com/PBandJaimie/701bb5633e06917f5ba5d4aa56c98468#file-2021-07-28-md

Reply:
I couldn't open the link :(
I think your way is much better to be honest. I just got one way to work. Lifestyle hooks ftw!